server/public/styles.css

```diff
@@ -0,0 +1,47 @@
+html,
+body {
+  margin: 0 auto;
+  padding: 0;
+  height: 100%;
+  max-width: 575px;
```

Comment:
Are these doing what you actually need? I don't really understand the max-width part. Just let the document flow.